Sources/URLRouting/Path.swift

```diff
@@ -20,25 +34,3 @@ extension Path: ParserPrinter where ComponentParsers: ParserPrinter {
     try self.componentParsers.print(output, into: &input)
   }
 }
-
-public struct PathComponent<ComponentParser: Parser>: Parser
```

Comment:
Moved this into `PathBuilder` since it isn't intended to be used directly and this cleans up the base docs.